easybuild/easyconfigs/s/SNAP-ESA/SNAP-ESA-9.0.0-Java-1.8.eb

```diff
@@ -0,0 +1,71 @@
+easyblock = 'Binary'
+
+name = 'SNAP-ESA'
+version = '9.0.0'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://step.esa.int/main/toolboxes/snap/'
+description = """
+The Sentinel Application Platform (SNAP) is a common architecture for all
+Sentinel Toolboxes being jointly developed by Brockmann Consult, SkyWatch and
+C-S.
+The SNAP architecture is ideal for Earth Observation processing and analysis
+due to the following technological innovations: Extensibility, Portability,
+Modular Rich Client Platform, Generic EO Data Abstraction, Tiled Memory
+Management, and a Graph Processing Framework."""
+
+toolchain = SYSTEM
+
+local_installer = 'esa-snap_all_unix_%s.sh' % version.replace('.', '_')
+local_varfile = 'SNAP-ESA-8.0-response.varfile'
+
+source_urls = [
+    'https://download.esa.int/step/snap/%(version_major_minor)s/installers',
+    'https://step.esa.int/downloads/%(version_major_minor)s/installers',
+]
+sources = [local_installer]
+patches = [(local_varfile, '.')]
+checksums = [
+    {'esa-snap_all_unix_9_0_0.sh': 'ad6ef8c679c80c58660fe23ab5fb4974f9cd3ff4b119ad8e7e40dc79dbe03896'},
+    {'SNAP-ESA-8.0-response.varfile': '5ea98e3376bb3df52b9c4c99ab4986015296b815021e16486b58e4aad58e21a4'},
+]
+
+# The installation is executed with the bundled JRE 1.8.0_242 (Zulu)
+# at runtime we switch it to AdoptOpenJDK, which is known to be more reliable for SNAP-ESA
+dependencies = [
+    ('Java', '1.8'),
```

Comment:
The current installation instructions say that it needs Java/11. Is it worth the upgrade?
See https://senbox.atlassian.net/wiki/spaces/SNAP/pages/10879039/How+to+build+SNAP+from+sources

Reply:
1.8 is also the old oracle variant we should avoid using and at least favor `('Java', '8'),`

Reply:
@lexming fixed in 8fbc32e6e2